src/Stratis.Features.FederatedPeg/PremineCoinbaseSplitter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NBitcoin;
+using Stratis.Features.FederatedPeg.Interfaces;
+
+namespace Stratis.Features.FederatedPeg
+{
+    public class PremineCoinbaseSplitter : ICoinbaseSplitter
+    {
+        /// <summary>
+        /// The number of outputs we break the premine reward up into, so that the federation can build more than one transaction at once.
+        /// </summary>
+        public const int FederationWalletOutputs = 10;
```

Comment:
Won't block over it because we do this everywhere, but this could be an injected value.